routers/__tests__/families.router.js

```diff
@@ -0,0 +1,96 @@
+const faker = require("faker");
+const jwt = require("jsonwebtoken");
+const config = require("../../config.env");
+const { ApiError } = require("../../helpers");
+
+const request = require("supertest");
+const { FamilieModel, UserModel } = require("../../database/modules");
+const UserServer = require("../../server");
+
+// const authCeck = require('../../middlewares/auth-check');
+
+// jest.mock('../../database/models/UserModel')
+// jest.mock('../../helpers/get-logger.js')
+
+const findOneSpy = jest.spyOn(UserModel, "findOne");
+const res = {
+  send: jest.fn(),
+  status: jest.fn().mockReturnThis()
+};
+
+describe("Acceptance tests suitcase example", () => {
```

Comment:
Acceptance already means we need to use real database requests. Mocks = unit test